CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Unreleased Changes
 2.11.419 (2019-12-18)
 ------------------
 
+* Feature - Support `s3_us_east_1_regional_endpoint` from `SharedConfig`.
```

Comment:
Not sure if this should be a new version or not. Gem has not been published for htis version.